### PR TITLE
shooter indicator bugfix 2

### PR DESCRIPTION
**Pull request by mondraker100**
When everybody AFK on server while playing shooter mode it changes map on start but it changes too quick that bugs shooter's indicator.

### CODE REVIEW

**Comment by AfuSensi** [comment]
Hi, thanks for your pull request.
However, i think this is the wrong way to fix this issue.
It should get fixed in https://github.com/MrGreenGaming/MTA-Resources/blob/master/resources/%5Brace%5D/race/modes/shooter.lua

My guess is that putting the call to stop the timebars, `clientCall(g_Root, 'sh_stopTimeBars')` inside the function `function Shooter:endMap()` will stop this bug from appearing.

**Comment by mondraker100** [comment]
> Hi, thanks for your pull request.
> However, i think this is the wrong way to fix this issue.
> It should get fixed in https://github.com/MrGreenGaming/MTA-Resources/blob/master/resources/%5Brace%5D/race/modes/shooter.lua
> 
> My guess is that putting the call to stop the timebars, `clientCall(g_Root, 'sh_stopTimeBars')` inside the function `function Shooter:endMap()` will stop this bug from appearing.

Thanks for the advice AfuSensi it worked, i'll be adapted on Mr.Green's functions within time :P

File: resources/[race]/race/modes/shooter.lua [hunk 1]
```diff
@@ -877,6 +877,7 @@ end
 function Shooter:endMap()
 	self:cleanup()
 	RaceMode.endMap(self)
+	setTimer(function() clientCall(g_Root, 'sh_stopTimeBars') end, 7000, 1) -- shooter indicator bugfix (Thanks to the AfuSensi for idea :P)
```
Comment (Haxardous) [inline resources/[race]/race/modes/shooter.lua:880]: no need for the comment :P

**Comment by Haxardous** [comment]
Will be closed for inactivity.

**Pull request closed without merging**